src/libs/actions/ReimbursementAccount/setupWithdrawalAccount.js

```diff
@@ -135,6 +137,58 @@ function mergeParamsWithLocalACHData(data) {
     return updatedACHData;
 }
 
+function getVBBADataForOnyx() {
+    return {
+        optimisticData: [
+            {
+                onyxMethod: CONST.ONYX.METHOD.MERGE,
+                key: ONYXKEYS.REIMBURSEMENT_ACCOUNT,
+                value: {
+                    isLoading: true,
+                    errors: null,
```

Comment:
This would clear out any existing errors, correct?

Reply:
Correct (on this key). We basically optimistically decided that no errors will happen